go.mod

```
@@ -12,7 +12,7 @@ replace (
 	github.com/docker/distribution => github.com/docker/distribution v2.8.2+incompatible
 	github.com/docker/docker => github.com/docker/docker v24.0.0-rc.2.0.20230801142700-69c9adb7d386+incompatible
 	github.com/docker/libnetwork => github.com/docker/libnetwork v0.8.0-dev.2.0.20190624125649-f0e46a78ea34
-	github.com/flannel-io/flannel => github.com/flannel-io/flannel v0.22.2
+	github.com/flannel-io/flannel => github.com/flannel-io/flannel v0.24.0
```

```suggestion
	github.com/flannel-io/flannel => github.com/flannel-io/flannel v0.24.2
```

Ref: https://github.com/k3s-io/k3s/blob/release-1.28/go.mod#L109

---

Probably fine since we don't use the embedded flannel anyways I guess.